autobahn/wamp_invocation.hpp

```diff
@@ -195,13 +200,13 @@ class wamp_invocation_impl
      * Reply to the invocation with positional arguments.
      */
     template <typename List>
-    void result(const List& arguments);
+    void result(const List& arguments, bool intermediate = false);
```

Comment:
Yep. This PR combines multiple things (and it's easier to merge focused PRs). And it's unclear to be too what `intermediate` is for ..